### PR TITLE
Add vueJS samples

### PR DESCRIPTION
**Pull request by alaameddeb**

### CODE REVIEW

File: portlet/vue-portlet-webpack/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        "plugin:vue/recommended",
+        "plugin:jest/recommended"
+    ],
+    "globals": {
+        "Vue": true,
+        "eXo": true,
+        "$": true,
+        "global": true,
+        "exoi18n": true,
+        "ExtensionRegistry": true
```
Comment (thomasdelhomenie) [inline portlet/vue-portlet-webpack/.eslintrc.json]: The ExtensionRegistry is not available yet, it should be added in the sample once it is integrated in the product IMO.

File: portlet/vue-portlet/pom.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<project>
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.acme.samples</groupId>
+	<artifactId>vue-sample</artifactId>
```
Comment (thomasdelhomenie) [inline portlet/vue-portlet/pom.xml]: Both portlets have the same artifacts. IMO we should make them unique to make sure there will be no conflict if both are built.
Same for the extension names.

Comment (alaameddeb) [reply portlet/vue-portlet/pom.xml]: Done

File: portlet/vue-portlet-webpack/.eslintrc.json (hunk 1 above)
Comment (alaameddeb) [reply portlet/vue-portlet-webpack/.eslintrc.json]: Done

**Pull request merged**